Add Tip to use ShowAddAccountAsync

azhan-msft opened the pull request:
Add a tip for callers to use ShowAddAccountAsync() instead of Show() if they want to query the status of the operation

azhan-msft committed:
Add Tip to use ShowAddAccountAsync

Add a tip for callers to use ShowAddAccountAsync() instead of Show() if they want to query the status of the operation

beneluxboy commented:
@msatranjr : The PR **passed** the acceptance checklist. Please evaluate further and approve/complete the PR if appropriate. Thanks!

azhan-msft commented:
@jwmsft Hey Jim, you reviewed&submitted my related PR: https://github.com/MicrosoftDocs/winrt-api/pull/796

Can you also take a look at this one?

Thanks,
-Andrew (azhan@microsoft.com)

msatranjr commented:
Please change URLs to site relative paths and remove en-us so that they will resolve to the correct locality.

azhan-msft committed:
Update Show ASP tip

Remove the en-us from the URL
Bold the link to math other links on this page
Change the phrasing of the sentence ot be less confusing

azhan-msft commented:
@msatranjr , I removed the en-us but I'm not sure what you mean by using "site relative paths". The URL I'm using looks the same as the other links on the page.

Can you show me an example of using site relative paths vs. absolute URLs? Should I update all the links on this page to follow that pattern?

azhan-msft commented:
@msatranjr  friendly ping on this PR. Hoping to get it in before EOY
